pkgs/development/libraries/signond/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitLab, pkgconfig, qmake, qtbase, qttools, libproxy, doxygen, graphviz }:
+
+stdenv.mkDerivation rec {
+  pname = "signond";
+  version = "8.60";
+
+  name = "${pname}-${version}";
+
```

Comment:
outputs?

Reply:
¿Qué?

Reply:
hah el español

How about, which appropriate outputs for a split package?